model/registry/hardware.yaml

```diff
@@ -0,0 +1,113 @@
+groups:
+  - id: registry.hardware
+    prefix: hw
+    type: attribute_group
+    brief: >
+      Attributes for hardware.
```

Comment:
It sounds the goal is to describe hardware component rather than a generic hardware, so I'd prefer to have `hw.component` or `hardware.component` as a namespace

Reply:
Can we change the whole namespace? It will go from current `hw.*` to `hw.XX.*` which means breaking changes for **all** fields

Reply:
I strongly disagree with this. In hardware, we're specifying metrics that apply to different levels: `hw.energy` may apply to a processor, a server, a UPS or even a data center. Same for temperature. Hardware does include components, but it's not just that, and it doesn't need to be namespaced as there are no collisions here.

Reply:
well, the `hw.id` seems to be better called `hw.component.id`.  While some things can stay in the `hw` root namespace, some might need more love to become descriptive